scripts/assumeRole.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# To run this script requires you to have the following environment variables set:
+# GDS_EMAIL_ADDRESS - the email address of the account for the 'gds-users' AWS account
+# ROLE_ARN - the ARN of the role you want to assume in the relevant AWS account
+: "${GDS_EMAIL_ADDRESS:?GDS_EMAIL_ADDRESS not set or empty}"
```

Comment:
When trying it out, I noticed that the error message includes the name of the variable anyway
`./assumeRole.sh:6: GDS_EMAIL_ADDRESS: GDS_EMAIL_ADDRESS not set or empty`

, handily, so I think you can leave it out of the message after the `?`

Reply:
Fixed!